src/main/java/com/redhat/red/build/koji/model/xmlrpc/messages/GetImageBuildResponse.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc.messages;
+
+import com.redhat.red.build.koji.model.xmlrpc.KojiImageBuildInfo;
+import org.commonjava.rwx.binding.anno.DataIndex;
+import org.commonjava.rwx.binding.anno.IndexRefs;
+import org.commonjava.rwx.binding.anno.Response;
+
+@Response
+public class GetImageBuildResponse
```

Comment:
Okay, I've fallen a little behind the times in this. KojiMavenBuildInfo is a supplemental class intended to query a different table in koji. The problem is groupId / artifactId / version is stored in a separate table, and we have to make a second query to get all the information needed to understand a Maven build. If this is similar for Windows and Image builds, then please proceed.